<launch>
  <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="bt_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>
  <node respawn="true" pkg="teleop_twist_joy"
        type="teleop_node" name="teleop" />
  </node>
</launch>
